### PR TITLE
update supporters block on dark theme

### PR DESCRIPTION
**Pull request by Arhell**
**What type of PR is this?**
fix block

Add one of the following kinds:
/kind bug
/kind design

**Which issue(s) this PR fixes**:
resolve https://github.com/karmada-io/website/issues/71

### CODE REVIEW

File: src/css/custom.css [hunk 1]
```diff
@@ -145,3 +148,7 @@ article header h1 {
 .title {
   font-size: 2rem;
 }
+
+html[data-theme='dark'] .support-wrapper li {
+  background-color: #ffffff;
+}
```
Comment (RainbowMango) [inline src/css/custom.css:154]: No newline at end of the file.

Comment (Arhell) [reply src/css/custom.css]: Done

**Comment by RainbowMango** [comment]
/approve

Leave LGTM to @rgrupesh

**Review by rgrupesh — APPROVED** [review]
/lgtm

**Pull request merged**